patterns/1-initial/include-product-owners.md

```diff
@@ -0,0 +1,72 @@
+## Title
+
+Include Product Owners
+
+## Patlet
+
+TBD
```

Comment:
Engaging and educating Product Owners about InnerSource can help them modify their actions (e.g., in the space of KPIs) to help InnerSource collaboration work better.

Reply:
Awesome, adding that in.